### PR TITLE
Mirrorconf blog potatocode

### PR DESCRIPTION
**Pull request by Sveijk**

### CODE REVIEW

**Comment by Sveijk** [comment]
Hi @nerdyvaishali 

Would you please check this PR if you have time? I managed to remove the old blogpost from 17 september, which has been published on 4th of October. Now that first blogpost is not in this PR anymore and the new blogpost is in it. Let me know if this is OK?

Kind regards, 
Sabine

File: blog/_posts/2018-10-04-Hurdles-of-a-beginning-developer.md [hunk 1]
```diff
@@ -51,4 +51,4 @@ Even so, the list of hurdles we encounter as junior developers grows bigger ever
 
 Just like we've found solutions to most of these, we're determined to chip at this list one issue at a time, and bungee jump as much as it takes :)
 
-Overall, we had a great smooth time at Rails Girls, with very few hiccups along the way because we were lucky to have our amazing supervisors and mentors. Going forward, I think we'll find that having a network to rely on will be one of the most effective ways to overcome these struggles. We can only hope that one day we'll be at a level where we can provide the same type of support for future aspiring devs!
```
Comment (v-thakkar) [inline blog/_posts/2018-10-04-Hurdles-of-a-beginning-developer.md]: @Sveijk This file is still here. :) Please remove it.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by Sveijk** [comment]
Hi @nerdyvaishali 

I think I fixed the issue now🙋‍♀️

File: blog/_posts/2018-11-03-potatoCode-at-MirrorConf.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+twitter: Sabin_E
+categories:
+  - student-posts
+---
+
+#### Team potatoCode at 🔮 _the future of the web_ 🔮 MirrorConf
+
+Asked by 🔮[MirrorConf](https://www.mirrorconf.com/) and Rails Girls Summer of Code to attend and speak on the conference in Braga, Portugal. We were excited and full of ideas: our talk would be about beginning devs and what hurdles they can encounter. To begin with, the organisation team did an excellent job in taking care of their speakers and because of this there was a very welcoming atmosphere. We felt very welcome! Our talk was due on the first day of the conference, among other great topics.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/uFBDLT7GmbA" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (v-thakkar) [inline blog/_posts/2018-11-03-potatoCode-at-MirrorConf.md]: Hi @Sveijk ! This video fails to load. Can you please fix that?

File: blog/_posts/2018-11-03-potatoCode-at-MirrorConf.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+<div class="image-credits"> Credits to Mirror Conf potatoCode 🙋‍♀🙋‍♀ Alina & Sabine at 0:44 seconds
+</div>
+
+To kickoff our talk, and out of curiosity, I asked the audience how many of them had just started programming about a year ago, some 30 hands were raised. Then I asked them who wasn't programming at all and another 50/60 hands were raised. So I concluded: "Awesome, I then assume all others are mid-level / senior developers, this talk will be for you!". Total attendees must have been around 250/300.
+
+<img src="/img/blog/2018/2018-10-23-potatoCode-at-MirrorConf-AlinaSabineTalk.jpg" alt="Alina & Sabine talk, hurdles of a beginning dev">
```
Comment (v-thakkar) [inline blog/_posts/2018-11-03-potatoCode-at-MirrorConf.md]: Also, all images fails to load as well. Can you please use markdown style to embed images?

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by Sveijk** [comment]
Hi @nerdyvaishali !
Could you check if this will work? I fixed all images to be in Markdown and made links to youtube videos.

Best regards🙋‍♀️

**Comment by v-thakkar** [comment]
@Sveijk Thanks, the blog post looks good now. I'll merge and publish the post today.

**Review by v-thakkar — APPROVED** [review]

**Comment by Sveijk** [comment]
Thank you very much Vaishali!
It's been a long time coming so glad all worked out fine now 🙋‍♀️

Best regards
Sabine

Op ma 26 nov. 2018 om 08:50 schreef vaishali Thakkar <
notifications@github.com>:

**Pull request merged**